run.py

```diff
@@ -500,7 +500,8 @@ def run(args):
                     config['add-repo'] = repo
             config['docker-insecure-registry'] = docker_insecure_registry
             config['skip_version_compare'] = skip_version_compare
-            if config and config.get('ansi_config'):
+            # if cdn installation fetch docker details from suite file
```

Comment:
@sunilangadi2, can you please comment where cdn details should reside and what variable to need to be set.